virtwho/virt/virt.py

```diff
@@ -492,6 +497,34 @@ def _get_data(self):
             reports[source_key] = report
         return reports
 
+    def _get_data_initial(self):
```

Comment:
This logic feels very similar to that of `DestinationThread._get_data`. What do you feel about abstracting the common bits out to a third method? Something like

```python
    def _get_data_common(self, source_keys):
        reports = {}
        for source_key in self.source_keys:
            report = self.source.get(source_key, NotSetSentinel)

            if report is None or report is NotSetSentinel:
                self.logger.debug("No report available for source: %s" %
                                  source_key)
                continue
            if report.hash == self.last_report_for_source.get(source_key, None):
                self.logger.debug('Duplicate report found, ignoring')
                continue
            reports[source_key] = report
        return reports
```

Then `_get_data` after the inital run logic branch can be simply `_get_data_common(self.source_keys)` and then `_get_data_initial` can contain only the waiting logic and the call to the common code... Thoughts?